src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs

```diff
@@ -463,52 +458,38 @@ private void WriteSetDiagnostics(Node node)
 
         private void WriteGreenAcceptMethods(Node node)
         {
-            //WriteLine();
-            //WriteLine("    public override TResult Accept<TArgument, TResult>(SyntaxVisitor<TArgument, TResult> visitor, TArgument argument)");
-            //WriteLine("    {");
-            //WriteLine("        return visitor.Visit{0}(this, argument);", StripPost(node.Name, "Syntax"));
-            //WriteLine("    }");
-            WriteLine();
-            WriteLine("    public override TResult Accept<TResult>(CSharpSyntaxVisitor<TResult> visitor) => visitor.Visit{0}(this);", StripPost(node.Name, "Syntax"));
             WriteLine();
-            WriteLine("    public override void Accept(CSharpSyntaxVisitor visitor) => visitor.Visit{0}(this);", StripPost(node.Name, "Syntax"));
+            WriteLine("public override void Accept(CSharpSyntaxVisitor visitor) => visitor.Visit{0}(this);", StripPost(node.Name, "Syntax"));
+            WriteLine("public override TResult Accept<TResult>(CSharpSyntaxVisitor<TResult> visitor) => visitor.Visit{0}(this);", StripPost(node.Name, "Syntax"));
         }
 
         private void WriteGreenVisitors()
         {
-            //WriteGreenVisitor(true, true);
-            //WriteLine();
-            WriteGreenVisitor(false, true);
-            WriteLine();
-            WriteGreenVisitor(false, false);
+            WriteGreenVisitor(withResult: true);
+            WriteGreenVisitor(withResult: false);
         }
 
-        private void WriteGreenVisitor(bool withArgument, bool withResult)
+        private void WriteGreenVisitor(bool withResult)
         {
             var nodes = Tree.Types.Where(n => !(n is PredefinedNode)).ToList();
 
             WriteLine();
-            WriteLine("  internal partial class CSharpSyntaxVisitor" + (withResult ? "<" + (withArgument ? "TArgument, " : "") + "TResult>" : ""));
-            WriteLine("  {");
-            int nWritten = 0;
+            WriteLine("internal partial class CSharpSyntaxVisitor" + (withResult ? "<TResult>" : ""));
+            OpenBlock();
             for (int i = 0, n = nodes.Count; i < n; i++)
             {
                 if (nodes[i] is Node node)
                 {
-                    if (nWritten > 0)
-                        WriteLine();
-                    nWritten++;
-
-                    WriteLine("    public virtual {0} Visit{1}({2} node{3}) => this.DefaultVisit(node{4});", withResult ? "TResult" : "void", StripPost(node.Name, "Syntax"), node.Name, withArgument ? ", TArgument argument" : "", withArgument ? ", argument" : "");
+                    WriteLine("public virtual {0} Visit{1}({2} node) => this.DefaultVisit(node);", withResult ? "TResult" : "void", StripPost(node.Name, "Syntax"), node.Name);
```

Comment:
boilerplate code like this removed all blank lines between members.  They didn't really help readability of the generated code.